docs/source/geo.rst

```diff
@@ -106,18 +106,27 @@ to :obj:`~simsopt.geo.SurfaceRZFourier` when running the code.
 The points :math:`\phi_j` and :math:`\theta_j` are used for evaluating
 the position vector and its derivatives, for computing integrals, and
 for plotting, and there are several available methods to specify these
-points.  For :math:`\theta_j`, you typically specify a keyword
-argument ``ntheta`` to the constructor when instantiating a surface
-class. This results in a grid of ``ntheta`` uniformly spaced points
-between 0 and 1, with no endpoint at 1. Alternatively, you can specify
+points. For :math:`\theta_j`,  you can specify
 a list or array of points to the ``quadpoints_theta`` keyword argument
-when instantiating a surface class, specifying the :math:`\theta_j`
-directly.  If both ``ntheta`` and ``quadpoints_theta`` are specified,
-an exception will be raised.  For the :math:`\phi` coordinate, you
+when instantiating surface classes.
+Similarly, you can pass a list or array to the
+``quadpoints_phi`` keyword argument of the constructor for any Surface
+subclass to specify the :math:`\phi_j` points directly.  
+
+Often, it is desirable to specify the number of grid points instead of  
+supplying to the Surface classes. For such cases, use the class method
```

Comment:
Missing words: "...instead of supplying quadpoints_phi and quadpoints_theta to the Surface..."